### DISCUSSION TITLE
Onboarding new hires

### DISCUSSION THREAD

**Discussion by bnaolu** [Bixal/rapid-response-team#95] [Ideas]
I attended this [InVision webinar](https://go.invisionapp.com/ctd-freehand-10-6-2021?utm_source=marketo&utm_medium=email&utm_content=&mkt_tok=MTg5LUlGSi02NDgAAAF_-goSJ1Xqq7qtB0P9efVrzlQMJfLS1a--gbI4NyebI4GX3ErTiIOi0xPUGMO7U5DRrj4Dkm80YXqQ71T1dTyq1EJ7yi0w5uG6Al-ChnC3PsE) around onboarding new hires and having a "single source of truth" for them. It showcased InVision's tool, "Freehand", but **I think we can recreate something either through GH Pages, Figma's FigJam, or even Miro**.

After our HCD Reorg retro yesterday (10/5), I felt like this was a space that's still pretty grey at Bixal and I think it's something that new hires crave (Nina G. was very outspoken about this and even I can say I've felt some confusion and loneliness along the way at the start).

Here are some notes I jotted down:

**Onboarding new hires at InVision**:
- Lots of documentation everywhere: Confluence, Google slides, Zoom recordings, etc…
- How can we make this fun and interactive?
- Discovered that new hires wanted clarity on their progress and managers wanted to have visibility to track and help new hires in this process.
- Contains useful tools and tips and main team channels
- Called their new hires “astronauts”

**New hire response**:
- Wasn’t sure what to expect especially with being remote and on an international team
- The personal welcome videos were a great way to experience the team "through those little pieces"
- Love that it was super interactive (being able to check things off, and the manager and “buddy” can see their progress)

**Comment by bnaolu** [comment]
I also remember ~~Demi mentioned~~ something about ~~Russell L. and Katharine~~ (I think it was Russell F. on the FED/DEA team?) working on creating something like this. But since they're on our delivery teams, maybe we can even expand on what they've started so they can focus on other priorities?

**Comment by pglevy** [comment]
This is a good one because it's a key part in the employee experience of transitioning from hiring to onboarding. And we have some joining in about a week, and hopefully others to follow soon.
